sources/VT100Screen.h

```diff
@@ -123,6 +123,10 @@ extern int kVT100ScreenMinRows;
    basedAtAbsoluteLineNumber:(long long)absoluteLineNumber
                       colors:(NSDictionary *)colors;
 
+- (void)linkTextInRange:(NSRange)range
+   basedAtAbsoluteLineNumber:(long long)absoluteLineNumber
+                        code:(unsigned short)code;
```

Comment:
`code` should be `URLCode` so that it's clear that his has to do with hyperlinks.